PKGBUILD

```diff
@@ -1,39 +1,25 @@
 # Maintainer: Pat Brisbin <pbrisbin@gmail.com>
-_gitname=vbump
-_gitroot='https://github.com/pbrisbin/vbump'
 
+_gitname=vbump
 pkgname="$_gitname-git"
-pkgver=0.0.0
+pkgver=r9.a229014
 pkgrel=1
 pkgdesc='Automatically bump package versions'
 arch=('any')
-url=$_gitroot
+url=https://github.com/pbrisbin/vbump
 license=('MIT')
+depends=('sh')
```

Comment:
Just curious about this. Is this not in the `base` or `base-devel` groups that can be assumed?

Reply:
I used namcap to check this package and it gave message `vbump-git E: Dependency bash detected and not included (programs ['sh'] needed in scripts ['usr/bin/vbump'])` so I just added it there to silence this error.
https://archlinux.org/packages/extra/any/namcap/